tests/test_connector.py

```diff
@@ -62,6 +62,17 @@ def send_my_event(self, event):
         with self.assertRaises(TypeError):
             MyConnector()
 
+    def test_event_subclasses(self):
+        class MyEvent(Message):
+            pass
+
+        class MyConnector(Connector):
+            @register_event(Message, include_subclasses=True)
+            def send_my_event(self, event):
+                pass
+
+        MyConnector({})
```

Comment:
Can we add some kind of assertion in here? Perhaps try sending an instance of `MyEvent` and assert that it doesn't raise.

Reply:
Addressed by @Cadair's suggested change, I think.